testing/testcafe/tests/editors/radioGroup/validationMessage.ts

```diff
@@ -0,0 +1,55 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import url from '../../../helpers/getPageUrl';
+import CheckBox from '../../../model/checkBox';
+import createWidget from '../../../helpers/createWidget';
+
+fixture`Radio Group ValidationMessage`
+  .page(url(__dirname, '../pages/t1020449.html'));
+
+test('message position is right in material (T1020449)', async (t) => {
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+  await t
+    .expect(await takeScreenshot('radiogroup-horizontal-validation.png', '#radio-form'))
+    .ok()
+    .expect(compareResults.isValid())
+    .ok(compareResults.errorMessages());
+});
+
+fixture`ValidationMessage`
+  .page(url(__dirname, '../../container.html'));
+
+test('ValidationMessage integrated in editor should not raise any errors when it is placed inside of form and has name "style" (T941581)', async (t) => {
+  const checkBox = new CheckBox('#container');
```

Comment:
shouldn't we use RadioGroup here? Now we test CheckBox inside of `radioGroup/validationMessage` package